src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -1,6 +1,13 @@
 
 package codeu.controller;
 
+import java.io.File;
```

Comment:
You've added imports, but it doesn't seem you've made changes that use these imports in the class. Are you planning to add in the changes later, or are these remnants of experimentation? If they are left over from trying these out, I suggest removing them, but I also do this all the time lol.